packages/@atjson/source-commonmark/test/extensions-test.ts

```diff
@@ -19,8 +19,8 @@ class MarkdownItSource extends CommonmarkSource {
 
 MarkdownItSource.defineConverterTo(OffsetSource, doc => {
   doc.where({ type: '-commonmark-s' }).set({ type: '-offset-strikethrough' });
-
-  return doc.convertTo(CommonmarkSource).convertTo(OffsetSource);
+  let convertCommonmark = getConverterFor(CommonmarkSource, OffsetSource);
+  return convertCommonmark(doc);
```

Comment:
I think the behaviour might be more predictable if the parent conversion happens first, rather than afterwards, just because the development cycle will be more like "start with the semi-converted document as per my base class, and then add on the extra conversions I need for this class."

I assume it doesn't make a difference here, but since it's one of our example cases, it might be good to set a good precedent? (Unless, of course, there's some reason why the parent conversion needs to happen at the end?)